src/main/scala/io/iohk/ethereum/consensus/validators/std/StdBlockValidator.scala

```diff
@@ -93,21 +142,16 @@ object StdBlockValidator extends BlockValidator {
   }
 
   /**
-   * This method allows validate that a BlockHeader matches a BlockBody. It only performs the following validations (stated on
-   * section 4.4.2 of http://paper.gavwood.com/):
-   *   - BlockValidator.validateTransactionRoot
-   *   - BlockValidator.validateOmmersHash
+   * This method allows validate that a BlockHeader matches a BlockBody.
    *
    * @param blockHeader to validate
    * @param blockBody to validate
    * @return The block if the header matched the body, error otherwise
    */
   def validateHeaderAndBody(blockHeader: BlockHeader, blockBody: BlockBody): Either[BlockError, BlockValid] = {
```

Comment:
I hate the method names in this class. This one says that it validates the header but it does not. There are several others like that. Maybe we should create a task to address it?

Reply:
I agree, I had some problems to understand what's going on there and what's a purpose because of naming. I will create a task.

Reply:
https://jira.iohk.io/browse/ETCM-171